src/app/logintest/page.tsx

```diff
@@ -0,0 +1,60 @@
+"use client"
```

Comment:
Great work saurabh, create  input fields, (label+input field), button, google button and facebook button as components. And then use components here.

Reply:
Also add primary and secondary colors in tailwind config file, so you don't have to type colors everywhere, and please take care of login design
1. input borders a little rounded
2. Bg color is white not yellow
3. Have a look at spacing
4. use google and facebook button from the design
5. Sign in should be in bold.

Take care of these little things.